test/link-changelog.bats

```diff
@@ -0,0 +1,62 @@
+setup() {
+    load 'test_helper/bats-support/load'
+    load 'test_helper/bats-assert/load'
+
+    DIR="$(cd "$( dirname "$BATS_TEST_FILENAME" )" >/dev/null 2>&1 && pwd)"
```

Comment:
In bash, `>word 2&>1` can be shortened to just `>&word` or `&>word` (apparently the latter of the two is preferred).

Reply:
Actually, I prefer to keep the longer form, because (1) it's POSIX compliant and (2) easier to read and understand, even after a few months (at least for me), when I usually forget these idioms.